ooiservices/app/uframe/data.py

```diff
@@ -7,13 +7,22 @@
 
 __author__ = 'Andy Bird'
 
+from flask import jsonify, request, current_app, url_for, Flask, make_response
+from ooiservices.app.uframe import uframe as api
+
 import numpy as np
 import calendar
 import time
 from dateutil.parser import parse
 from datetime import datetime
-
+from ooiservices.app.main.errors import internal_server_error
 from ooiservices.app import cache
+import requests
+
+#ignore list for data fields
+FIELDS_IGNORE = ["stream_name","quality_flag"]
+COSMO_CONSTANT = 2208988800
```

Comment:
@lukecampbell for sure, im not removing it as i do it on the plotting side. i did as a note to the get data function that the data units are `seconds since 1900` https://github.com/asascience-open/ooi-ui-services/pull/219/files#diff-d49ddecb2a01f4df8e82a6bba2377b3bR81